### PR TITLE
Feature/commands

### PR DESCRIPTION
**Pull request by KillaMelone01**

### CODE REVIEW

File: sales.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+   link: 'http://foo.bar.baz',
```
Comment (mjkatgithub) [inline sales.txt]: remove this file

File: bot.js [hunk 2]
```diff
@@ -40,32 +42,61 @@ mongoose.connect(
 mongoose.Promise = global.Promise;
 
 
-
-
 //Execute Commands
-bot.on('message', message => {
-  if(message.author.equals(bot.user)) return;
-  if(!message.content.startsWith(PREFIX)) return;
+bot.on("message",function(message) {
```
Comment (mjkatgithub) [inline bot.js]: let's use Arrow functions
```suggestion
bot.on("message", message => {
```

File: bot.js [hunk 3]
```diff
@@ -40,32 +42,61 @@ mongoose.connect(
...
+  if (message.author.equals(bot.user)) return;
+  console.log(message.content);// gives you a output , what is writen in the Chat
```
Comment (mjkatgithub) [inline bot.js]: pls don't log all messages

File: bot.js [hunk 4]
```diff
@@ -40,32 +42,61 @@ mongoose.connect(
...
+  if(!message.content.startsWith(PREFIX)) return;//return if there is no Prefix
+  var args = message.content.substring(PREFIX.length).split(" ");
```
Comment (mjkatgithub) [inline bot.js]: let's use: let
https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var#11444416

File: bot.js [hunk 5]
```diff
@@ -40,32 +42,61 @@ mongoose.connect(
...
+    message.channel.send(Today)
+    break;
+    case "test"://just for testing purpose(sends message)
+      message.channel.send("test done");
+      break;
 
-  switch(cmd) {
-    case 'foo':
-      //dumy data
+    case "testembed"://just for testing purpose(sends embed)
```
Comment (mjkatgithub) [inline bot.js]: remove testembed case
testing is done

File: bot.js [hunk 6]
```diff
@@ -40,32 +42,61 @@ mongoose.connect(
...
+    case "addsale"://ads a Sale to the List
+      var SaleLink = args[1]// here schould be the link to the sale
+      var SaleEnd = args[2]
+      var SaleDescription = args[3]// here schould be the description to the sale
+      message.channel.send(SaleLink+SaleEnd+SaleDescription)
       let saleData = {
-        link: "http://foo.bar.baz",
+        link: SaleLink,
         start: Date("now"),
         end: Date("now"),
```
Comment (mjkatgithub) [inline bot.js]: use SaleEnd

File: controllers/sales.js [hunk 7]
```diff
@@ -1,15 +1,27 @@
...
+        .setTitle(doc.description)
+        .addField("link",doc.link)
+        .addField("until:", doc.end)
+        this.message.channel.send(Salesembed);
+    });
   })
   .catch(console.error)
 }
 
+function sendSale(item, index) {
```
Comment (mjkatgithub) [inline controllers/sales.js]: remove this function

File: bot.js [hunk 8]
```diff
@@ -40,32 +42,61 @@ mongoose.connect(
...
+  if (message.author.equals(bot.user)) return;
+  console.log(message.content);// gives you a output , what is writen in the Chat
+  if(!message.content.startsWith(PREFIX)) return;//return if there is no Prefix
+  var args = message.content.substring(PREFIX.length).split(" ");
 
-  msgArray = message.content.substring(PREFIX.length).split(' ');
-  let cmd = msgArray[0].toLowerCase();
+  //all Commands
+  switch (args[0].toLowerCase()) {
+    case "day":
```
Comment (mjkatgithub) [inline bot.js]: remove day case

**Review by mjkatgithub — CHANGES_REQUESTED** [review]

File: sales.txt (hunk 1 above)
Comment (NYC00kie) [reply sales.txt]: done

File: bot.js (hunk 2 above)
Comment (NYC00kie) [reply bot.js]: done

File: bot.js (hunk 4 above)
Comment (NYC00kie) [reply bot.js]: done

File: bot.js (hunk 5 above)
Comment (NYC00kie) [reply bot.js]: done

**Pull request merged**